src/services/evaluation-administration-service.ts

```diff
@@ -537,9 +537,12 @@ export const publish = async (id: number) => {
   )
 }
 
-export const reopen = async (id: number) => {
+export const reopen = async (id: number, eval_end_date: Date | null) => {
   const evaluationAdministration = await EvaluationAdministrationRepository.getById(id)
 
+  if (eval_end_date != null && new Date() > new Date(eval_end_date)) {
```

Comment:
`eval_end_date` should never be null in this case so when it is reopened and user passes a null `eval_end_date`, it should return an error. If they don't want to change the end date but they reopened, user is allowed to not pass any value so `eval_end_date` is undefined and not null (meaning no changes in the db).

Basically we should return an error if eval_end_date is null. We could return "Invalid end date"

Reply:
done